diff -aurN gedit-plugins-2.15.3.orig/configure.ac gedit-plugins-2.15.3/configure.ac
--- gedit-plugins-2.15.3.orig/configure.ac	2006-06-11 00:36:33.000000000 +0200
+++ gedit-plugins-2.15.3/configure.ac	2006-06-11 15:28:17.436222720 +0200
@@ -283,7 +283,7 @@
 # ================================================================
 # Plugin specific checks
 # ================================================================
-function check_plugin_defined() {
+check_plugin_defined() {
 	if echo "$2" | egrep "\b($1)\b" > /dev/null;
 	then
 		return 1
@@ -292,17 +292,17 @@
 	fi
 }
 
-function plugin_defined() {
+plugin_defined() {
 	check_plugin_defined "$1" "$PLUGINS"
 	return $?
 }
 
-function plugin_defined_explicit() {
+plugin_defined_explicit() {
 	check_plugin_defined "$1" "$explicit_plugins"
 	return $?
 }
 
-function undef_plugin() {
+undef_plugin() {
 	if test -z "$disabled_plugins"
 	then
 		disabled_plugins="$1 ($2)"
