diff -urN gedit-plugins-2.19.90/configure.ac gedit-plugins-2.19.90.new/configure.ac
--- gedit-plugins-2.19.90/configure.ac	2007-09-03 21:46:09.000000000 +0200
+++ gedit-plugins-2.19.90.new/configure.ac	2007-09-08 16:31:22.000000000 +0200
@@ -50,8 +50,8 @@
 PKG_CHECK_MODULES(GEDIT, [
 	glib-2.0 >= 2.8.0
 	gtk+-2.0 >= 2.8.0
-	gtksourceview-1.0 >= 1.2.0
-	gedit-2.16 >= 2.16.0
+	gtksourceview-2.0 >= 1.90.4
+	gedit-2.16 >= 2.19.91
 	libglade-2.0 >= 2.4.0
 	gnome-vfs-2.0 >= 2.13.4
 ])
